### PR TITLE
fix(perf-charts): headers

### PR DESCRIPTION
**Pull request by doralchan**
**Before:**
Copy Transaction
Screen Shot 2020-05-01 at 8 53 49 AM
Performance Landing
Screen Shot 2020-04-30 at 3 42 45 PM
Transaction Summary 1
Screen Shot 2020-04-30 at 3 42 32 PM
Transaction Summary 2
Screen Shot 2020-04-30 at 3 42 17 PM
Discover V2 Proof
Screen Shot 2020-04-30 at 3 46 19 PM
Releases V2 Proof
Screen Shot 2020-04-30 at 3 46 30 PM

**After:**
Copy Transaction
Screen Shot 2020-05-01 at 11 01 44 AM
Performance Landing
Screen Shot 2020-04-30 at 3 37 09 PM
Transaction Summary 1
Screen Shot 2020-04-30 at 3 37 17 PM
Transaction Summary 2
Screen Shot 2020-04-30 at 3 37 26 PM
Discover V2 Proof
Screen Shot 2020-04-30 at 3 37 39 PM
Releases V2 Proof
Screen Shot 2020-04-30 at 3 38 11 PM

### CODE REVIEW

File: src/sentry/static/sentry/app/utils/discover/styles.tsx [hunk 1]
```diff
@@ -75,21 +75,22 @@ export const StyledShortId = styled(ShortId)`
...
   return (
     <React.Fragment>
+      {shortId}
       <Clipboard value={value}>
-        <span>
-          <StyledIconCopy size="xs" />
-        </span>
+        <IconWrapper>
+          <IconCopy size="xs" />
+        </IconWrapper>
```
Comment (markstory) [inline src/sentry/static/sentry/app/utils/discover/styles.tsx:92]: One problem with putting the icon on the right, is that the cell actions in discover rows will overlap this icon and make it hard to click it. Maybe we don't need the copy icon at all? Generally people will be able to use cell actions to update search conditions, or the event id will be a link as it is in performance views.

File: src/sentry/static/sentry/app/views/performance/charts/index.tsx [hunk 2]
```diff
@@ -141,25 +139,4 @@ class Container extends React.Component<Props> {
...
-  border-color: transparent;
-  margin-bottom: 0;
-`;
-
-const HeaderContainer = styled('div')`
-  display: grid;
-  grid-template-columns: 1fr 1fr;
-  grid-gap: ${space(2)};
-  padding: ${space(2)} ${space(1.5)} ${space(1)} ${space(1.5)};
-`;
```
Comment (markstory) [inline src/sentry/static/sentry/app/views/performance/charts/index.tsx:163]: :clap:

File: src/sentry/static/sentry/app/utils/discover/styles.tsx (hunk 1 above)
Comment (doralchan) [reply src/sentry/static/sentry/app/utils/discover/styles.tsx:92]: Ohhh! good to know I'll revert it

Comment (doralchan) [reply src/sentry/static/sentry/app/utils/discover/styles.tsx]: Actually yeah - let's remove the copy icon after all...

**Review by markstory — APPROVED** [review]

**Pull request merged**